### PR TITLE
{astro,lib}[GCCcore/10.2.0] CFITSIO v3.49, HEALPix v3.70

### PR DESCRIPTION
**Pull request by jose-d**
Healpix 3.70 with patch solving this bug: https://sourceforge.net/p/healpix/bugs/113/ .

(created using `eb --new-pr`)

### CODE REVIEW

**Comment by jose-d** [comment]
Test report by @jose-d
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in total)
koiosvm3 - Linux centos stream 8, x86_64, Intel Xeon Processor (Skylake, IBRS), Python 3.6.8
See https://gist.github.com/10fe31048499397c86b7e07f634e5d5f for a full test report.

**Review by Micket — APPROVED** [review]
lgtm

**Comment by Micket** [comment]
Test report by @Micket
**FAILED**
Build succeeded for 1 out of 2 (2 easyconfigs in total)
vera-c1 - Linux centos linux 7.9.2009, x86_64, Intel Xeon Processor (Skylake), Python 2.7.5
See https://gist.github.com/bfde1c3f8502f60500ec9045a7c67101 for a full test report.

**Comment by boegel** [comment]
@boegelbot please test @ generoso

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 1 out of 2 (2 easyconfigs in total)
node2605.swalot.os - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz (haswell), Python 3.6.8
See https://gist.github.com/b932b09bc1a1304cc64e9dfd5db205bf for a full test report.

**Comment by arielzn** [comment]
@jose-d 

I was needing this and pulled, I've noticed on my side `sanity_checks_paths` for HEALPix were failing for the static libraries checks.

It seems `libhealpix.a` is actually built  with on the `make test` part called on `sanity_check_commands`.

I've moved the `make test` to postinstall commands and then all was built fine and tests passed:

```diff
--- HEALPix-3.70-GCCcore-10.2.0.eb.orig 2021-06-02 16:30:20.225645262 +0200
+++ HEALPix-3.70-GCCcore-10.2.0.eb      2021-06-02 16:31:25.204633131 +0200
@@ -42,8 +42,9 @@
     'HEXE': '%(installdir)s/bin'
 }
 
+postinstallcmds = [ 'cd %(installdir)s && make test &> build/testresults.txt' ]
+
 sanity_check_commands = [
-    "cd %(installdir)s && make test &> build/testresults.txt",
     'test $(grep -c "test completed" %(installdir)s/build/testresults.txt) -eq 4',
     'test $(grep -c "normal completion" %(installdir)s/build/testresults.txt) -eq 10',
 ]
```

Is a bit weird why the test report was passing before on Generoso, is it possible it used another eb version in which sanity_check_commands and sanity_check_paths are executed on different order ?

**Comment by jose-d** [comment]
> Is a bit weird why the test report was passing before on Generoso

thanks for looking at this, @arielzn ! I had no idea where this nondeterminism is coming from. I directly edited the file here at GitHub, will double-check it at my system and hopefully we can have this in next release.

**Comment by jose-d** [comment]
Test report by @jose-d
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in total)
n6 - Linux centos stream 8, x86_64, Intel(R) Xeon(R) Gold 6130 CPU @ 2.10GHz, Python 3.6.8
See https://gist.github.com/f5bfb23c49347efac04c011892db3437 for a full test report.

File: easybuild/easyconfigs/h/HEALPix/HEALPix-3.70-GCCcore-10.2.0.eb [hunk 1]
```diff
@@ -0,0 +1,68 @@
+easyblock = 'ConfigureMake'
```
Comment (boegel) [inline easybuild/easyconfigs/h/HEALPix/HEALPix-3.70-GCCcore-10.2.0.eb:1]: @jose-d Any reason for not using the custom easyblock for HEALPix which we have already?

That would simplify the easyconfig file quite a bit...

See https://github.com/easybuilders/easybuild-easyconfigs/blob/develop/easybuild/easyconfigs/h/HEALPix/HEALPix-3.50-GCCcore-8.2.0.eb

**Review by boegel — CHANGES_REQUESTED** [review]

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in total)
node2612.swalot.os - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz (haswell), Python 3.6.8
See https://gist.github.com/3e0a9123a4f37edd6e0abdca797a8f3a for a full test report.

**Pull request closed without merging**